qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -217,30 +217,121 @@ def convert_acquire(self, shift, instruction):
                 })
         return self._qobj_model(**command_dict)
 
-    def convert_single_acquires(self, shift, instruction,
-                                qubits=None, memory_slot=None, register_slot=None):
-        """Return converted `AcquireInstruction`, with options to override the qubits,
-        memory_slot, and register_slot fields. This is useful for grouping
-        AcquisitionInstructions which are operated on a single AcquireChannel and
-        a single MemorySlot.
+    def convert_bundled_acquires(
+            self,
+            shift,
+            instructions_,
```

Comment:
why `instructions_`?

Reply:
instructions is is already in the namespace as a module. It could be `instrs` if you prefer.

Reply:
definitely a downside to doing that :(

Reply:
I could change the module import to `instructions_` or `instrs`?